src/localization-editor.scss

```diff
@@ -10,4 +10,11 @@ $block: #{$fd-namespace}-localization-editor;
 
     padding: fd-space(2);
   }
+
+  // Elements
+  &__list {
+    li {
```

Comment:
Due to the nature of self contained styles, we can't use element selectors to prevent bleeding. This either needs a different selector or new class.

Reply:
I will add some class then.

Reply:
It's done by other PR.